### PR TITLE
added Solar System Model

### PR DESCRIPTION
**Pull request by Asymtode712**
Closes: #102 

- **Title** :  Solar System Model
- **Name:** : Siddheya Kulkarni
- **Idenitfy yourself:** : GSSoC'24 Contributor

## Description
This PR implements a 3D interactive solar system visualization using React and Three.js. It allows users to explore the planets in our solar system, view their relative sizes and positions, and access basic information about each celestial body.

## Key Features
- 3D rendering of the Sun and planets using Three.js
- Interactive orbit controls for user navigation
- Planet selection and information display
- Realistic textures for each planet
- Scaling logic to balance visual appeal with relative accuracy

## Components

- `SolarSystem.js`: Manages the arrangement and rotation of planets
- `Planet.js`: Renders individual planets with textures and rotation
- `PlanetInfo.js`: Displays information about the selected planet

### How Has This Been Tested? ⚙️
It has been tested on my local machine

### Video

https://github.com/Avdhesh-Varshney/WebMasterLog/assets/115717746/ebdae7ea-98ff-4637-93c8-1fdc077b9dd0

### Type of change ☑️

What sort of change have you made:

- [X] New feature (non-breaking change which adds functionality)

### Checklist: ☑️

- [X] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [X] This PR does not contain plagiarized content.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly wherever it was hard to understand.
- [X] My changes generate no new warnings.

### Note to reviewers 📄

@Avdhesh-Varshney please review and merge this PR

### CODE REVIEW

File: React-JS-Projects/Advanced/solar-system-model/package-lock.json
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Advanced/solar-system-model/package-lock.json:1]: remove this only package.json is sufficient.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@Asymtode712 capitalize the first letter of folder name.

**Comment by Asymtode712** [comment]
@Avdhesh-Varshney Done with the changes and resolved the merge conflict

**Comment by Asymtode712** [comment]
@Avdhesh-Varshney Remaned the folder.....please check the PR now and really sorry for making such mistakes....won't repeat them in future🙏

**Review by Avdhesh-Varshney — APPROVED** [review]
@Asymtode712 PR Approved 🎉

**Pull request merged**